### PR TITLE
Internalisation

### PR DESCRIPTION
**Pull request by MattBinnie**

### CODE REVIEW

File: Assets/Scripts/GlobalValues/WordStore.cs [hunk 1]
```diff
@@ -5,10 +5,17 @@ static class WordStore
 {
     private static List<string> WORDS;
     private static readonly TextAsset WORDTEXTFILE = Resources.Load<TextAsset>("WordStore/WordList");
+    private static readonly TextAsset FRENCHWORDTEXTFILE = Resources.Load<TextAsset>("WordStore/FrenchWordList");
```
Comment (AlanRoth) [inline Assets/Scripts/GlobalValues/WordStore.cs:8]: bad practice to load in both resources when you don't use both

File: Assets/Scripts/GlobalValues/WordStore.cs [hunk 2]
```diff
@@ -5,10 +5,17 @@ static class WordStore
...
 
-    public static void GenerateWords()
+    public static void GenerateWords(string language)
     {
-        WORDS = new List<string>(WORDTEXTFILE.text.Replace("\r\n", "\n").Split('\n'));
+        if(language == "English")
```
Comment (AlanRoth) [inline Assets/Scripts/GlobalValues/WordStore.cs:12]: If we change the word list name to something like "WordList_English.txt" and "WordList_French.txt" we could probably do something like 
```
private static string PATH_PREFIX = "WordStore/WordList_"
private static string FILE_EXT = ".txt"

public static void GenerateWords(string language) {
    var filePath = PATH_PREFIX + language + FILE_EXT;
    TextAsset WordList = Resources.Load<TextAsset>(filePath);
    WORDS = new List<string>(WordList .text.Replace("\r\n", "\n").Split('\n'));
}
```

Obviously with some extra checks for if the language is null or if the file does not exist to load in a default file instead or something like that.

With this all we have to do is drop in a wordlist file then add the language in options and it should be loaded in, with the weights missing

Comment (MattBinnie) [reply Assets/Scripts/GlobalValues/WordStore.cs:12]: Yeah think i tried to explain this, this way we would just have to make sure we save the list in a consistent way, i can impliment this

File: Assets/Scripts/GlobalValues/WordStore.cs [hunk 3]
```diff
@@ -4,11 +4,13 @@
...
-    private static readonly TextAsset WORDTEXTFILE = Resources.Load<TextAsset>("WordStore/WordList");
+    private static readonly string PATH_PREFIX = "WordStore/WordList_";
 
-    public static void GenerateWords()
+    public static void GenerateWords(string language)
     {
-        WORDS = new List<string>(WORDTEXTFILE.text.Replace("\r\n", "\n").Split('\n'));
+        var filePath = PATH_PREFIX + language;
+        TextAsset WordList = Resources.Load<TextAsset>(filePath);
+        WORDS = new List<string>(WordList.text.Replace("\r\n", "\n").Split('\n'));
```
Comment (AlanRoth) [inline Assets/Scripts/GlobalValues/WordStore.cs:13]: Does it work?

**Review by AlanRoth — APPROVED** [review]

File: Assets/Scripts/GlobalValues/WordStore.cs (hunk 3 above)
Comment (MattBinnie) [reply Assets/Scripts/GlobalValues/WordStore.cs:13]: no

**Pull request merged**